Rename Valid language tag to Known primary language tag (#1738)
act-rules/act-rules.github.io: _rules/element-lang-matches-default-language-off6ek.md

- the element is an [inclusive descendant][] in the [flat tree][] of a `body` element; and
- the element is in a [document][] with a [content type][] of `text/html`; and

> - the element's `lang` [attribute value][] has a [known primary language tag][]; and

Reviewer: I like how it turns out. "**has** a primary tag" does emphasize nicely the partiality of the match 👍

## Assumptions

> - This rule assumes that user agents and assistive technologies can programmatically determine [known primary language tags][known primary language tag] even if these do not conform to the [RFC 5646][] syntax.

Reviewer: ```suggestion
- This rule assumes that user agents and assistive technologies can programmatically determine [known primary language tags][known primary language tag] even on [language tags][language tag] who do not conform to the [RFC 5646][] syntax.
```
(must be done on all rules)

> - This rule assumes that only [known primary language tags][known primary language tag] are enough to satisfy [Success Criterion 3.1.2 Language of Parts][sc312]; this notably excludes [grandfathered tags][] and [ISO 639.2][] three-letters codes, both having poor support in assistive technologies.

Reviewer: ```suggestion
- This rule assumes that only[language tags][language tag] with a [known primary language tag][] are enough to satisfy [Success Criterion 3.1.2 Language of Parts][sc312]; this notably excludes [grandfathered tags][] and [ISO 639.2][] three-letters codes, both having poor support in assistive technologies.
```
(must be done on all)

#### Inapplicable Example 6

> The `lang` [attribute value][] of this `p` element is an [iso 639.2][] three letters code. This is not a [known primary language tag][].

Reviewer: ```suggestion
The `lang` [attribute value][] of this `p` element is an [iso 639.2][] three letters code. This has no [known primary language tag][].
```
Or maybe go fully "Its primary tag is `eng`, which is not a known primary tag" 🤔

Author: Not sure I like that. How about this:

```suggestion
The `lang` [attribute value][] of this `p` element has no [known primary language tag][] because the `eng`[iso 639.2][] three letters code does not exist in the [language subtag registry][].
```

#### Inapplicable Example 7

> The `lang` [attribute value][] of this `p` element is a [grandfathered tag][grandfathered tags]. This is not a [known primary language tag][].

Reviewer: ```suggestion
The `lang` [attribute value][] of this `p` element is a [grandfathered tag][grandfathered tags]. This has no [known primary language tag][].
```

> The `lang` [attribute value][] of this `p` element is a [grandfathered tag][grandfathered tags]. This is not a [known primary language tag][].

Author: ```suggestion
The `lang` [attribute value][] of this `p` element has no [known primary language tag][] because the `i-lox` [grandfathered tag][grandfathered tags] does not exist in the [language subtag registry][].
```

[text inheriting its programmatic language]: #text-inheriting-language 'Definition of Text Inheriting its Programmatic Language from an Element'
[sc312]: https://www.w3.org/TR/WCAG21/#language-of-parts 'Success Criterion 3.1.2 Language of Parts'
[usc312]: https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts.html 'Understanding Success Criterion 3.1.2: Language of Parts'

> [known primary language tag]: #known-primary-language-tag 'Definition of Known Primary Language Tag'

Author: ```suggestion
[known primary language tag]: #known-primary-language-tag 'Definition of Known Primary Language Tag'
[language subtag registry]: http://www.iana.org/assignments/language-subtag-registry/language-subtag-registry
```
